src/Components/CommentCard.jsx

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import { useEffect } from "react";
+import { useState } from "react";
+import axios from "axios";
+import { FaTimes } from "react-icons/fa";
+
+const CommentCard = ({ review_id }) => {
+  const [comments, setComments] = useState([]);
+  const id = review_id;
+  const [err, setErr] = useState(null);
+
+  useEffect(() => {
+    axios
+      .get(`https://games-nc.herokuapp.com/api/reviews/${id}/comments`)
+      .then((res) => {
+        setComments(res.data.comments);
+      });
+  }, [id]);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    let comment = e.target.firstElementChild.value;
+    let commentObj = {
+      body: comment,
+      comment_id: Math.random(),
+      author: "tickle122",
+    };
+    setComments([commentObj, ...comments]);
+    document.getElementById("newcomment").value = "";
+
```

Comment:
You shouldn't be using any `document.get*` DOM functions in React!

Please refactor this to be a controlled component instead

Reply:
Also you shouldn't need to generate your own `comment_id` here - the back end should do this job for you!